Header header
nav_msgs/Odometry odom
geometry_msgs/Point tip
geometry_msgs/Vector3 euler_angles
geometry_msgs/Vector3 euler_angles_velocity
bool is_point_contact
geometry_msgs/Point contact_point